lib/index.js

```diff
@@ -25,3 +25,4 @@ exports.createWindow = function(html) {
 };
 
 exports.impl = require('./impl');
+exports.HTMLParser = HTMLParser;
```

Comment:
`exports` generally has only things which are actually part of the HTML spec.  For non-standard methods (and I believe `HTMLParser` is an unstandardized interface) it should live someplace under `exports.impl`, probably `exports.impl.HTMLParser`.